### PR TITLE
Vet clinic database: add join table for visits.

### PR DESCRIPTION
**Pull request by habasm**
### Description and objectives
- In this project, I added some many-to-many relationships and write more complex queries to use them to answer questions.
- I manipulated the different types of relationships between tables.
- Prepared complex queries that answer analytical questions.
- Used primary key & foreign key mechanism for joining tables
### The following screenshots shows sample SQL codes and query results
image
image
image
image
image
image
image
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -30,4 +30,26 @@ create sequence animal_id_seq;
...
 ALTER TABLE animals ADD COLUMN species_id INTEGER REFERENCES species(id);
-ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
+-- project four--
+CREATE TABLE vets
+(
+    id SERIAL  primary key not null,
+    name text ,
```
Comment (CollinsTatang) [inline schema.sql:39]: - [ ] Please change the name data type to `VARCHAR(255)` so that you can be able to limit the maximum length of inserting a string.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @habasm, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

- [x] Please add screenshots for all your query results to the Pull Request Description. I notice you added some of the queries without the result.

**Optional**
- [ ] Please remove the `Live Demo` section from your README file since this project does not require the deploy link.

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (habasm) [reply schema.sql:39]: @CollinsTatang fixed.

**Comment by habasm** [comment]
@CollinsTatang I fixed, but I am not able to accept a comment on README. because from the README it says that "no live preview". And nothing problem in the read me. And from the screenshot I added other shots, and I accept the comment.

Thanks, @habasm

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @habasm, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "bug fix".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by CollinsTatang** [comment]
> @CollinsTatang I fixed, but I am not able to accept a comment on README. because from the README it says that "no live preview". And nothing problem in the read me. And from the screenshot I added other shots, and I accept the comment.
> 
> Thanks, @habasm

Hi @habasm 
Great job implementing the requested changes, for the README it was optional sorry I didn't indicate it.

**Pull request merged**